projects/aca-shared/src/lib/components/info-drawer/info-drawer.component.scss

```diff
@@ -0,0 +1,20 @@
+@use '../../styles/mat-selectors' as ms;
+
+.aca-info-drawer {
```

Comment:
We use Encapsulation.None so we have rule to wrap our styles in component's selector, so please wrap all these styles inside: 

aca-info-drawer {

}

selector